Dockerfile

```diff
@@ -1,37 +1,48 @@
-FROM memgraph/memgraph:latest AS memgraph-mage
-
-FROM memgraph-mage
+FROM debian:buster as debian
 USER root
 
-RUN apt-get update && \
-    apt-get --yes install curl git cmake g++ clang python3-dev uuid-dev && \
-    cd /usr/local/bin && \
-    ln -s /usr/bin/python3 python && \
-    rm -rf /var/lib/apt/lists/* /tmp/* /var/tmp/*
-
-# Install Rust
-RUN curl https://sh.rustup.rs -sSf | sh -s -- -y
-ENV PATH="/root/.cargo/bin:${PATH}"
-
-# Set mage as working directory
-WORKDIR /mage
-COPY . /mage
+RUN apt-get update && apt-get install -y \
+    build-essential `mage` \
+    cmake           `mage` \
+    curl            `mage-memgraph` \
+    g++             `mage` \
+    git             `mage` \
+    libcurl4        `memgraph` \
+    libpython3.7    `memgraph` \
+    libssl1.1       `memgraph` \
+    openssl         `memgraph` \
+    python3         `mage-memgraph` \
+    python3-pip     `mage-memgraph` \
+    --no-install-recommends \
+    && rm -rf /var/lib/apt/lists/* /tmp/* /var/tmp/*
+
+#Memgraph
+RUN pip3 install networkx==2.4 numpy==1.19.2 scipy==1.5.2 \
+    && curl https://download.memgraph.com/memgraph/v1.6.1/debian-10/memgraph_1.6.1-community-1_amd64.deb --output memgraph.deb \
```

Comment:
The MG version should be a parameter, and not hardcoded

Reply:
Of course, didn't forget about it, just still under development.